package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rif-relay-client",
-  "version": "1.0.0-alpha.1",
+  "version": "1.0.0-alpha.2",
```

Comment:
If it is a hot fix should it not be 
```suggestion
  "version": "1.0.1-alpha.1",
```
?

Reply:
no "stable" version has been released, so I think we should still move forward the next pre-release number instead of the minor one

Reply:
ok